packages/bot-web-ui/src/components/main-content/toolbox.scss

```diff
@@ -2,9 +2,10 @@
     $toolbox: &;
     $category-text-spacing: 10px;
 
-    height: var(--bot-content-height) !important;
-    top: 10.4rem !important;
     width: 25.6rem !important;
+    position: absolute;
+    top: 0 !important;
```

Comment:
now that we are rewriting the components, is there any possibility to get rid if `!important` here?